source/MaterialXRenderGlsl/GlslRenderer.h

```diff
@@ -61,7 +61,7 @@ class MX_RENDERGLSL_API GlslRenderer : public ShaderRenderer
     /// required for program validation and rendering.
     /// An exception is thrown on failure.
     /// The exception will contain a list of initialization errors.
-    void initialize() override;
+    void initialize(SharedContextHandle sharedcontext = nullptr) override;
```

Comment:
This looks like a good approach for the C++ API, but I believe we'll still need to declare these new default values in our Python bindings, so that clients can continue to call the method without arguments:

```
.def("initialize", &mx::GlslRenderer::initialize, py::arg("sharedContext") = nullptr)
```